contracts/access/manager/AccessManager.sol

```diff
@@ -63,6 +64,21 @@ contract AccessManager is Context, Multicall, IAccessManager {
         _;
     }
 
+    /**
+     * @dev Check that the caller is an admin and that the top-level function currently executing has been scheduled
+     * sufficiently ahead of time, if necessary according to admin delays.
+     */
+    modifier onlyDelayedAdmin() {
```

Comment:
The name implies that the operation is necessarily delayed. How would you feel about just calling that
```solidity
modifier onlyAdmin() {
```